package.json

```diff
@@ -10,18 +7,28 @@
     "type": "git",
     "url": "git@github.com:cssinjs/jss.git"
   },
-  "keywords": [
```

Comment:
keywords gone?

Reply:
ah I see, they are in the jss package

Reply:
Yeah, I removed some unnecessary things in the root package.json